setup.py

```diff
@@ -80,7 +80,7 @@ def get_static_file_paths():
 
 setup(
     name='toucan_connectors',
-    version='3.5.1',
+    version='3.5.1.1',
```

Comment:
```suggestion
    version='3.5.2',
```

3.5.1.1 is invalid semver